linode/k8s-node-decorator
# Implement the container app; add helm chart templates; add release automation; add build CI

#### Testing approach
- Apply all manifest to a LKE cluster, `kubectl apply -f /path/to/your/manifests`
- Verify the pod is ok, `kubectl get pods`, you should see one decorator pod for each node.
- Verify the app is running correctly by checking the log, `kubectl logs YOUR_POD_NAME`
- See the result of label updates of nodes, `kubectl get nodes -o wide --show-labels`

#### Generate manifest from Helm chart
- `helm template helm/k8s-node-decorator > my_manifest.yaml`
- `kubectl apply -f my_manifest.yaml`
